panther_controller/README.md

```diff
@@ -1,122 +1,13 @@
-[//]: # (ROS_API_NODE_COMPATIBLE_1_2)
-
-[//]: # (ROS_API_NODE_NAME_START)
-
-### imu_broadcaster
-
-[//]: # (ROS_API_NODE_NAME_END)
-
-[//]: # (ROS_API_NODE_DESCRIPTION_START)
-
-External node type:*[imu_sensor_broadcaster/imu_sensor_broadcaster](https://github.com/ros-controls/ros2_controllers/tree/master/imu_sensor_broadcaster)*.
-
-The broadcaster to publish readings of IMU sensors.
-
-[//]: # (ROS_API_NODE_DESCRIPTION_END)
-
-#### Publishers
-
-[//]: # (ROS_API_NODE_PUBLISHERS_START)
-
-- `imu/data` [*sensor_msgs/msg/Imu*]: data from IMU sensor.
-
-[//]: # (ROS_API_NODE_PUBLISHERS_END)
-[//]: # (ROS_API_NODE_END)
-
-[//]: # (ROS_API_NODE_START)
-
-[//]: # (ROS_API_NODE_COMPATIBLE_1_0)
-[//]: # (ROS_API_NODE_COMPATIBLE_1_2)
-
-[//]: # (ROS_API_NODE_NAME_START)
-
-### joint_state_broadcaster
-
-[//]: # (ROS_API_NODE_NAME_END)
-
-[//]: # (ROS_API_NODE_DESCRIPTION_START)
-
-External node type:*[joint_state_broadcaster/joint_state_broadcaster](https://github.com/ros-controls/ros2_controllers/tree/master/joint_state_broadcaster)*.
-
-The broadcaster reads all state interfaces and reports them on specific topics.
-
-[//]: # (ROS_API_NODE_DESCRIPTION_END)
-
-#### Publishers
-
-[//]: # (ROS_API_NODE_PUBLISHERS_START)
-
-- `dynamic_joint_states` [*control_msgs/msg/DynamicJointState*] - provides information about the state of various movable joints in a robotic system.
-- `joint_states` [*sensor_msgs/msg/JointState*] - provides information about the state of various joints in a robotic system.
-
-[//]: # (ROS_API_NODE_PUBLISHERS_END)
-[//]: # (ROS_API_NODE_END)
-
-[//]: # (ROS_API_NODE_START)
-
-[//]: # (ROS_API_NODE_COMPATIBLE_1_0)
-[//]: # (ROS_API_NODE_COMPATIBLE_1_2)
-
-[//]: # (ROS_API_NODE_NAME_START)
-
-### drive_controller
-
-[//]: # (ROS_API_NODE_NAME_END)
-
-[//]: # (ROS_API_NODE_DESCRIPTION_START)
-
-External node type:*[diff_drive_controller/diff_drive_controller](https://github.com/ros-controls/ros2_controllers/tree/master/diff_drive_controller)*.
-
-Controller which manages mobile robots with a differential drive. It converts velocity commands for the robot body into wheel commands for the base. It also calculates odometry from hardware feedback and shares it.
-
-[//]: # (ROS_API_NODE_DESCRIPTION_END)
-
-#### Subscribers
-
-[//]: # (ROS_API_NODE_SUBSCRIBERS_START)
-
-- `cmd_vel` [*geometry_msgs/msg/Twist*]: command linear and angular velocity values.
-
-[//]: # (ROS_API_NODE_SUBSCRIBERS_END)
-
-#### Publishers
-
-[//]: # (ROS_API_NODE_PUBLISHERS_START)
+## Launch Files
 
-- `/tf` [*tf2_msgs/msg/TFMessage*]: tf tree. Published only if `enable_odom_tf=true`
-- `odometry/wheels` [*nav_msgs/msg/Odometry*]: odometry data from wheel encoders.
+- `controller.launch.py`: Establishes communication with the hardware by loading the robot's URDF with plugins and configures the controllers to exchange information between the engine driver and the IMU.
 
-[//]: # (ROS_API_NODE_PUBLISHERS_END)
-[//]: # (ROS_API_NODE_END)
+## Configuration Files
```

Comment:
In some readme files (like here) `Configuration Files` are section (##) but in others subsection of the `Launch Files` section (###). Please standardize it across packages.